src/NuGet/EntityFramework/tools/EntityFramework6.psm1

```diff
@@ -1205,7 +1205,11 @@ function GetConfigPath($project)
         $configFileName = 'app.config'
     }
 
-    return GetProperty $project.ProjectItems.Item($configFileName).Properties 'FullPath'
```

Comment:
Note, new csproj returned null if the item didn't exist, but old csproj threw. Missed this since the .NET Framework Console template includes an app.config